docs/public/SUMMARY.md

```diff
@@ -0,0 +1,110 @@
+- [Installation]()
```

Comment:
Is there a good reason why this is not README.md so that it shown by default by Github?

Reply:
An error. This has been fixed.